td.vue/src/main.desktop.js

```diff
@@ -182,6 +182,26 @@ window.electronAPI.onSaveModelFailed((_event, fileName, message) =>  {
     app.$toast.warning(message);
 });
 
+//adding shortcut registration from main process
+window.addEventListener(`keydown`, (event) => {
```

Comment:
Will this work, considering we are not using the Electron file menu role, but defining our own one?